pkg/splunk/enterprise/configuration.go

```diff
@@ -246,27 +246,27 @@ func prepareSplunkSmartstoreConfigMap(identifier, namespace string, crKind strin
 // getSplunkPorts returns a map of ports to use for Splunk instances.
 func getSplunkPorts(instanceType InstanceType) map[string]int {
 	result := map[string]int{
-		"splunkweb": 8000,
-		"splunkd":   8089,
+		"http-splunkweb": 8000,
```

Comment:
Can we add string templates for the names of the ports in names.go? Intention would be to maintain a single place in case of further changes too.